cli/src/action/keygen.rs

```diff
@@ -145,8 +145,7 @@ pub fn create_key_pair(
             .open(private_key_path.as_path())
             .map_err(|err| CliError::EnvironmentError(format!("{}", err)))?;
 
-        private_key_file
-            .write(private_key.as_hex().as_bytes())
+        writeln!(&private_key_file, "{}", private_key.as_hex())
```

Comment:
This needs to be `&mut`

Reply:
Not sure if that's the case. The build did fail because I messed up a rebase but it should be fixed now.